app/components/FormikInput/InputComponent.tsx

```diff
@@ -24,6 +24,7 @@ type InputComponentType = {
   placeholder?: string;
   type?: string;
   value?: string | number;
+  disabled: boolean;
```

Comment:
`disabled` prop for inputs should be optional because otherwise every time you use that component you will need to specify a value for this prop explicitly, even if it would be false.

Type `FormikInputType` in `app/components/FormikInput/index.tsx` should also be updated with the new prop